webdriver-spec.html

```diff
@@ -5451,13 +5464,14 @@
     and <code>meta</code> all set to <code>false</code>.
 
     <p>A <a>pointer input source</a> has a corresponding <dfn>pointer
-    input state</dfn> object. This consists of a <code>subtype</code>
-    property, which has the possible
-    values <code>"mouse"</code>, <code>"pen"</code>,
-    and <code>"touch"</code>, a <code>primary</code> property which is
-    a boolean, a pressed property which is a set of integers
-    an <code>x</code> property which is an unsigned integer and
-    a <code>y</code> property which is an unsigned integer.
+        input state</dfn> object. This consists of
+        a <code>subtype</code> property, which has the possible
+        values <code>"mouse"</code>, <code>"pen"</code>,
+        and <code>"touch"</code>, a <code>primary</code> property
```

Comment:
Is this the exhaustive list of `subtype` properties? What if more are somehow added to pointer events in later revisions of that spec? Or a UA-specific subtype?